### ISSUE TITLE
TypeError: can't convert undefined to object after creating and selecting a model.

### ISSUE THREAD

**Issue by lucasdavid** [megagonlabs/ruler#2]
I get the following error after creating the model "hello":
```
TypeError: can't convert undefined to object
```
![image](https://user-images.githubusercontent.com/1595030/100387415-cfd96500-3006-11eb-9299-08a07b004cfd.png)

Any idea why this is happening? :(

python --version 3.6.9
node -v v13.7.0

Requirements:
```
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.5/en_core_web_sm-2.2.5.tar.gz
h5py==2.10.0
Keras-Preprocessing==1.1.2
networkx==2.5
nltk==3.5
numpy==1.17.3
pandas==0.24.2
scikit-learn==0.21.3
scipy==1.4.1
spacy==2.2.3
tensorboard==2.4.0
tensorboard-plugin-wit==1.7.0
tensorboardX==1.9
tensorflow==2.3.0
tensorflow-estimator==2.3.0
torch==1.1.0
...
```

**Comment by lucasdavid** [comment]
So I managed to fix this by adding the `labels` entry in the `server/models/{MODEL}/labels.json` file, but now I having a second problem:
![image](https://user-images.githubusercontent.com/1595030/100615248-188a6a00-32f6-11eb-9576-7c26204805ba.png)

The stacktrace printed is:
![image](https://user-images.githubusercontent.com/1595030/100615289-26d88600-32f6-11eb-9046-033e5c617ef9.png)

I was only able to get to this page by clicking in the "labelling task" in the previous one. Clicking in the last "next" button would collapse step 4, display the message "all done" but would not redirect me to this page. I was wondering if this might be the culprit.

**Comment by lucasdavid** [comment]
Hey Sara. Thank you for your reply.

As my first test, I'm using the first 340 rows of the [consumer complaint](https://www.kaggle.com/sebastienverpile/consumercomplaintsdata)) (kaggle) dataset. I also renamed the column `Consumer complaint narrative` to `text` and 
the `issue` col to `label`.

I've uploaded it once again. Clicking over the "next" btn a few times does seem to pop the progress bar/navigate to the home.
![image](https://user-images.githubusercontent.com/1595030/100907499-27fde480-34a9-11eb-8e7f-45651b8771b6.png)
